packages/client/README.md

```diff
@@ -221,7 +221,8 @@ client.addHandler(ProviderEvents.Error, async (eventDetails) => {
 
 ### Shutdown:
 
-The OpenFeature API provides a close function to perform a cleanup of all providers attached to any client.
+The OpenFeature API provides a close function to perform a cleanup of all registered providers.
+This should only be called when your application is in the process of shutting down.
```

Comment:
@toddbaert @lukas-reining I updated the text here so that we could release a patch version. It's a bit of a workaround but I think the additional context is valuable. What do you think?